src/components/Audio/Audio.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faVolumeUp, faSpinner } from "@fortawesome/free-solid-svg-icons";
+
+import "./Audio.scss";
+
+interface LocalProps {
+    url: string;
+}
+type Props = LocalProps;
+
+interface LocalState {
+    play: boolean;
+}
+
+class AudioComponent extends React.Component<Props, LocalState> {
+    private audioElem: HTMLAudioElement;
+    constructor(props) {
+        super(props);
+        this.state = {
+            play: false
+        };
+        this.audioElem = new Audio(this.props.url);
+        this.audioElem.onended = this.stop;
```

Comment:
typo or is this function actually not camel case?

Reply:
This is the way it is defined.